variables.tf

```diff
@@ -132,3 +132,9 @@ variable "hieradata_repo_user" {
 variable "control_vshn_net_token" {
   type = string
 }
+
+variable "team" {
+  type        = string
+  description = "Team to assign the load balancers to in Icinga. All lower case."
+  default     = ""
```

Comment:
I think for this Terraform we should make the team variable mandatory as there's no existing setups to consider. The whole Terraform is VSHN-specific anyway, so it's not reducing usability for other people anymore.